package.json

```diff
@@ -43,20 +43,20 @@
     "postcss-preset-env": "^6.7.0",
     "postcss-safe-parser": "^4.0.1",
     "regenerator-runtime": "^0.13.3",
-    "sass-loader": "7.3.1",
-    "style-loader": "^1.0.0",
+    "sass-loader": "^7.3.1",
+    "style-loader": "^1.1.2",
     "terser-webpack-plugin": "^2.3.4",
-    "webpack": "^4.41.3",
+    "webpack": "^4.41.5",
```

Comment:
https://github.com/webpack/webpack/releases/tag/v4.42.0

Reply:
Hey @gauravtiwari, does it make sense to support this PR? Maybe it would be better to break it into smaller ones?

Reply:
Thanks @pustovalov Yep it does. I was just concerned if there is going to be any breaking changes? I guess everything looks good except file loader node version requirements? https://github.com/webpack-contrib/file-loader/releases/tag/v5.0.0

Thanks for working on this and sorry about the delay.

Reply:
you are right, looks like breaking change only in `file-loader`